tests/sentry/runner/commands/test_configoptions.py

```diff
@@ -208,6 +208,48 @@ def test_bad_sync(self):
 
         assert ConsolePresenter.ERROR_MSG % ("set_on_disk_option", "option_on_disk") in rv.output
 
+    def test_sync_unset_options(self):
+
+        # test options set on disk with and without prioritize disk, tracked
+        # and not tracked
+        # test options set on db, verify that untracked options are properly # deleted
+
+        rv = self.invoke(
+            "-f",
+            "tests/sentry/runner/commands/unsetsync.yaml",
```

Comment:
You could put this yaml file with other test data in `fixtures`. It could be helpful to have an `options` directory for this and other test fixture files.